grpc-gcp/src/main/java/com/google/cloud/grpc/GcpManagedChannel.java

```diff
@@ -776,29 +760,49 @@ public int getMaxActiveStreams() {
    */
   protected ChannelRef getChannelRef(@Nullable String key) {
     if (key == null || key.equals("")) {
-      return pickLeastBusyChannel(false);
+      return pickLeastBusyChannel();
     }
-    ChannelRef channelRef = affinityKeyToChannelRef.get(key);
-    if (channelRef == null || !fallbackEnabled) {
-      return channelRef;
+    ChannelRef mappedChannel = affinityKeyToChannelRef.get(key);
+    if (mappedChannel == null || !fallbackEnabled) {
+      return mappedChannel;
```

Comment:
nit: return null;

Reply:
I'm sorry, this condition is not easy to read, but there is an "or" in the condition, so the mappedChannel might be not null here too.

Reply:
ah yes. I need to read this code again. I am a bit confused now